packages/xrpc-server/src/rate-limiter.ts

```diff
@@ -109,6 +110,30 @@ export class RateLimiter implements RateLimiterI {
       }
     }
   }
+
+  delete = async (
```

Comment:
I might call this `reset` rather than `delete` since `delete` is a keyword. i also find `reset` to be a bit more understandable (which you use in `server.ts`)